Main/ClientApp/src/Authorization/Login.js

```diff
@@ -56,9 +56,14 @@ export class Login extends Component {
     } else {
       switch (action) {
         case LoginActions.Login:
-          return <MessageWithLoader text="Processing login" />;
+          return <PanelWideMessage text="Processing login" throbber={true} />;
```

Comment:
I'm pretty sure if it is a boolean value, we don't have to put ={true} at all. Using this, we could change the name of the value to "withThrobber", and using this component would look like:
<PanelWideMessage withThrobber text=...>
Nothing wrong with this approach, but I like this flow better. Consider this.